.prettierrc.json

```diff
@@ -0,0 +1,21 @@
+{
+    "printWidth": 80,
+    "tabWidth": 2,
+    "useTabs": false,
+    "semi": false,
```

Comment:
Is this config used to format `.js` as well?

Our [house style](https://github.com/Agoric/agoric-sdk/wiki/Coding-Style) follows Airbnb, which _does_ include semicolons.

See also https://github.com/Agoric/agoric-sdk/tree/master/packages/eslint-config

Reply:
This is just for markdown files